src/NUnitFramework/framework/nunit.framework-netstandard13.project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
That is really weird, I don't get any warnings on initialize, build or run, but I do if I attempt to upgrade to 1.6.1. I was reticent to upgrade to 1.6.1 because we haven't tested that with the Visual Studio adapter, but i don't know if that is a problem. I'm also not seeing the warnings in the AppVeyor build, https://ci.appveyor.com/project/CharliePoole/nunit/build/3.7.0-ci-03975-pr-2174